oh_staff_ui/urls.py

```diff
@@ -17,4 +17,5 @@
     path("upload_file/<int:item_id>", views.upload_file, name="upload_file"),
     path("order_files/<int:item_id>", views.order_files, name="order_files"),
     path("browse/", views.browse, name="browse"),
+    path("oai/", views.oai, name="oai")
```

Comment:
Since the logic for this is all in the view, please add comment(s) here to document that `/oai/` needs to be called with `verb` and optionally `ark` parameters.